roles/multiqc/tasks/main.yml

```diff
@@ -4,15 +4,29 @@
   shell: "{{ ngi_pipeline_venv }}/bin/pip install multiqc --upgrade"
```

Comment:
Should multiqc from pypi be installed at the same time as multiqc from the github repo?